todo/src/components/TodoForm.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { connect } from "react-redux";
+import { addTodo } from "../actions";
+
+class TodoForm extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      inputField: ""
```

Comment:
With only one item in state, this would be a super smooth transition to using a functional component and Hooks instead of a class component 😎